ambry-network/src/main/java/com.github.ambry.network/NetworkClient.java

```diff
@@ -93,6 +93,7 @@ public NetworkClient(Selector selector, NetworkConfig networkConfig, NetworkMetr
   public List<ResponseInfo> sendAndPoll(List<RequestInfo> requestInfos)
       throws IOException {
     long startTime = time.milliseconds();
+    logger.trace("Initiating Send and poll at " + startTime);
```

Comment:
I am wondering how useful this log info would be. If everything is normal, the averaged send and poll time would be recorded by `networkClientSendAndPollTime` so there is no need to look into this log. If some particular send and poll takes abnormal time, it is hard to debug that send and poll operation since only time is logged. To me, this trace info would not be providing more information than the corresponding metric.